webdriver-spec.html

```diff
@@ -4105,7 +4108,46 @@
 
  <li><p><a>Scroll into view</a> the <var>element</var>.
 
- <li><p class=issue>Interact with the element.
+ <li><p>Run the <a>focusing steps</a> for the <a>element</a>.
+
+ <li><p>Set the cursor position to after the last character in the element
```

Comment:
This doesn't feel like a good definition of where the cursor is placed. I'm at a bit of a loss for figuring out a good alternative. Because we need to imply either the end of the element's Value or the end of the Text in the element (for contenteditable elements).